db/migrate/20181025230931_add_comments_to_vms_table.rb

```diff
@@ -0,0 +1,102 @@
+    change_column_comment :vms, :cpu_limit, "The CPU utilization limit for the VM."
+    change_column_comment :vms, :cpu_reserve, "Amount of CPU that is guaranteed available for the VM."
+
+    change_column_comment :vms, :cpu_reserve_expand, "Amount the CPU can grow beyond its normal limits if there "\
+                                                     "are unreserved resources available."
+
+    change_column_comment :vms, :cpu_shares, "The number of CPU shares allocated. Used to determine resource allocation "\
+                                             "in case of resource contention."
+
+    change_column_comment :vms, :cpu_shares_level, "CPU shares allocation level."
+    change_column_comment :vms, :created_on, "The timestamp the VM was added to the app inventory."
+    change_column_comment :vms, :deprecated, "Indicates whether or not a template is deprecated."
+    change_column_comment :vms, :description, "A description of the VM that is typically set by the provider, but may also be edited."
+    change_column_comment :vms, :ems_created_on, "The timestamp the VM or Templated was created on the EMS itself."
+    change_column_comment :vms, :ems_id, "ID of the ExtManagementSystem associated with the VM."
+    change_column_comment :vms, :evm_owner_id, "ID of VM owner from product database."
+    change_column_comment :vms, :ems_ref, "A native unique reference for the VM within the EMS."
+    change_column_comment :vms, :ems_ref_obj, "A YAML serialized version of the ems_ref."
+    change_column_comment :vms, :ems_cluster_id, "ID of associated cluster."
+
+    change_column_comment :vms, :fault_tolerance, "Indicates whether or not the VM is fault tolerant, i.e. if one host "\
+                                                  "crashes it will immediately pick up on another host in the cluster."
+
+    change_column_comment :vms, :flavor_id, "ID of associated flavor."
+    change_column_comment :vms, :format, "The format of the VM's disk, such as VMDK."
+    change_column_comment :vms, :guid, "A value used to uniquely identify the VM internally."
+    change_column_comment :vms, :id, "The internal database ID."
+    change_column_comment :vms, :host_id, "ID of the host that the VM/Template resides on."
+    change_column_comment :vms, :last_perf_capture_on, "Timestamp for last performance metrics capture."
+    change_column_comment :vms, :last_scan_on, "Timestamp for last SmartState analysis."
+    change_column_comment :vms, :last_scan_attempt_on, "Timestamp for last attempt to perform SmartState Analysis."
+    change_column_comment :vms, :last_sync_on, "Timestamp for last SmartState analysis."
+
+    change_column_comment :vms, :linked_clone, "Indicates whether the VM is a linked clone, i.e its disk is a referencing "\
+                                               "disk pointing to snapshot of another VM."
+
+    change_column_comment :vms, :location, "The location or region of the VM."
```

Comment:
For infra VMs this is the storage location e.g. `[datastore 1] vm/vm.vmx` if you want to include an example

Reply:
@agrare For now it's just set to "The location of the VM, or its underlying storage." Is that alright?